### DISCUSSION TITLE
关于使用开源，可商用的open llama重现本项目模型的可行性

### DISCUSSION THREAD

**Discussion by jeff31415** [ymcui/Chinese-LLaMA-Alpaca#382] [Ideas]
https://github.com/openlm-research/open_llama
近期这个开源的llama复现效果挺好的（而且兼容使用llama模型的方案），而且很有可能在结束训练后获得比原版更强的性能。目前有训练到400B的预览版本，挺接近原版的了。可以考虑在这个模型基础上把项目重现一次，这样方案就可以彻底开源了

**Comment by tianjiqx** [comment]
是的，时候有考虑吗

**Reply by jeff31415** [reply]
有别人做了这个了，不过性能怎么样不清楚

**Reply by tianjiqx** [reply]
嗯，有项目链接吗
